src/components/Song/SongPresentation.js

```diff
@@ -36,7 +46,9 @@ function SongPresentation({ entry }) {
               {entry?.titleTranslation}
             </h2>
             <div className="text-fv-charcoal-light">
-              {entry?.author?.length > 0 ? ` by ${entry.author}` : ''}
+              {entry?.acknowledgement?.length > 0
+                ? ` by ${entry.acknowledgement}`
+                : ''}
```

Comment:
Now that this field is not the author, this ui probably needs a little update. Maybe just replace "by" with "Acknowledgement: " ? @gmcauliffe Do you have opinions?